### PR TITLE
Dump logs for all services to Github after running e2e integration tests

### PR DESCRIPTION
**Pull request by wimax-grapl**
### Which issue does this PR correspond to?
#231 

### What changes does this PR make to Grapl? Why?
- Introduce a new script that just collects all `docker logs` for a given dobi `compose=` task
- Run that new script after an e2e test run
- Upload results of script to Github artifacts - learn more here! https://docs.github.com/en/free-pro-team@latest/actions/guides/storing-workflow-data-as-artifacts

### How were these changes tested?
by submitting this very diff!!!

### CODE REVIEW

File: etc/ci_scripts/dump_compose_logs.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/python3
+
+from typing import Any, List
+from tempfile import TemporaryDirectory
+from pathlib import Path
+import os
+import subprocess
+
+
+def _go_to_grapl_repo_root() -> None:
```
Comment (wimax-grapl) [inline etc/ci_scripts/dump_compose_logs.py:10]: in retrospect this is not needed

File: .github/workflows/grapl-build.yml [hunk 2]
```diff
@@ -130,3 +131,19 @@ jobs:
...
         run: |
           GRAPL_RELEASE_TARGET=debug TAG=latest ./dobi-linux --no-bind-mount run-e2e-integration-tests
+            
+      # always() allows this to run even if 'run tests' failed
+      - name: 'Collect e2e test logs'
+        if: ${{ always() }}
+        run: |
+          ./etc/ci_scripts/dump_compose_logs.py --compose-project "grapl-integration-tests"
+
+      # always() allows this to run even if 'run integration tests' failed
```
Comment (wimax-grapl) [inline .github/workflows/grapl-build.yml:141]: oh, make this match the above comment

File: etc/ci_scripts/dump_compose_logs.py (hunk 1 above)
Comment (jgrillo-grapl) [reply etc/ci_scripts/dump_compose_logs.py:10]: should probably remove it?

**Review by jgrillo-grapl — APPROVED** [review]
LGTM

**Pull request merged**